### PR TITLE
UUID power gen; config option to generate powers based off of player's UUID

### PR DESCRIPTION
**Pull request by Roseplate**
I also snuck in a change to the readme (V_4_Vicegod to Roseplate)

### CODE REVIEW

File: src/main/java/com/legobmw99/allomancy/modules/powers/CommonEventHandler.java [hunk 1]
```diff
@@ -45,6 +45,21 @@ public static void onAttachCapability(final AttachCapabilitiesEvent<Entity> even
             event.addCapability(AllomancerCapability.IDENTIFIER, provider);
         }
     }
+    
+    public static int randomNumbGen(Player player) {
```
Comment (legobmw99) [inline src/main/java/com/legobmw99/allomancy/modules/powers/CommonEventHandler.java:49]: I would prefer if this logic all just got moved inside the conditional. It doesn't get reused anywhere, so I don't think it needs to be a function

File: src/main/java/com/legobmw99/allomancy/modules/powers/CommonEventHandler.java [hunk 2]
```diff
@@ -45,6 +45,21 @@ public static void onAttachCapability(final AttachCapabilitiesEvent<Entity> even
...
+
+        char m = player.getUUID().toString().charAt(0);
+
+        String e = Character.toString(m);
+
+        int i = HexToDecimal(e);
+        return i;
+    }
+
+    public static int HexToDecimal(String s){
```
Comment (legobmw99) [inline src/main/java/com/legobmw99/allomancy/modules/powers/CommonEventHandler.java:59]: Same with this

**Review by legobmw99 — CHANGES_REQUESTED** [review]
Thanks for putting this together. I'd appreciate if you could change these, more about style than anything

**Comment by Roseplate** [comment]
done and done, I think, not super used to github

**Comment by legobmw99** [comment]
Thanks!

**Pull request merged**